Ports/SDL2/package.sh

```diff
@@ -11,7 +11,7 @@ configopts=(
     "-DPULSEAUDIO=OFF"
     "-DJACK=OFF"
     "-DSDL_LIBSAMPLERATE=OFF" # Disabled to prevent potential collision with host libsamplerate
-    "-DEXTRA_LDFLAGS=-laudio;-liconv;-ldl"
```

Comment:
While you are here, can you try removing ldl? I think it is never needed on Serenity as it's basically an alias for LibC.

Reply:
Still seems to work without it, I changed the commit to remove it.